docs/api/v3.rst

```diff
@@ -472,140 +522,230 @@ Build triggering
+    Retrieve a list of all sub-projects for a project.
 
     **Example request**:
 
     .. sourcecode:: bash
 
-        $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/users/
+        $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/subprojects/
 
     **Example response**:
 
     .. sourcecode:: json
 
         {
-            "username": "humitos",
-            "created": "2008-10-23T18:12:31+00:00",
-            "last_login": "2010-10-23T18:12:31+00:00"
+            "count": 25,
+            "next": "/api/v3/projects/pip/subprojects/?limit=10&offset=10",
+            "previous": null,
+            "results": ["PROJECT"]
         }
 
-    :>json string username: username for the user.
-    :>json string created: date and time when the user was created.
-    :>json string last_login: date and time for last time this user was logged in.
+    :>json integer count: total number of projects.
+    :>json string next: URI for next set of projects.
+    :>json string previous: URI for previous set of projects.
+    :>json array results: array of ``project`` objects.
 
     :requestheader Authorization: token to authenticate.
 
 
-User listing
-++++++++++++
+Translations
+~~~~~~~~~~~~
+
+Translations listing
+++++++++++++++++++++
 
 
-.. http:get:: /api/v3/projects/(str:project_slug)/users/
+.. http:get:: /api/v3/projects/(str:project_slug)/translations/
 
-    Retrieve list of users for a project.
+    Retrieve a list of all translations for a project.
 
     **Example request**:
 
     .. sourcecode:: bash
 
-        $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/users/
+        $ curl -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/pip/translations/
 
     **Example response**:
 
     .. sourcecode:: json
 
         {
             "count": 25,
-            "next": "/api/v3/projects/pip/users/limit=10&offset=10",
+            "next": "/api/v3/projects/pip/translations/?limit=10&offset=10",
             "previous": null,
-            "results": ["USER"]
+            "results": ["PROJECT"]
         }
 
-    :>json integer count: total number of users.
-    :>json string next: URI for next set of users.
-    :>json string previous: URI for previous set of users.
-    :>json array results: array of ``user`` objects.
+    :>json integer count: total number of projects.
+    :>json string next: URI for next set of projects.
+    :>json string previous: URI for previous set of projects.
+    :>json array results: array of ``project`` objects.
 
     :requestheader Authorization: token to authenticate.
 
 
-Subprojects
-~~~~~~~~~~~
+Redirects
+~~~~~~~~~
 
-Subprojects listing
-+++++++++++++++++++
+Redirects allow the author to redirect an old URL of the documentation to a new one.
+This is useful when pages are moved around in the structure of the documentation set.
+See :doc:`/user-defined-redirects` for more information.
```

Comment:
We should probably put a similar description for all the resources (eg. Translations)